src/sentry/search/events/base.py

```diff
@@ -120,6 +122,11 @@ def _resolve_user_display_alias(self, _: str) -> SelectType:
         columns = ["user.email", "user.username", "user.ip"]
         return Function("coalesce", [self.column(column) for column in columns], USER_DISPLAY_ALIAS)
 
+    def _resolve_transaction_status(self, _: str) -> SelectType:
+        return Function(
+            "toUInt64", [self.column(TRANSACTION_STATUS_ALIAS)], TRANSACTION_STATUS_ALIAS
```

Comment:
if you check the type in clickhouse this is a UInt8 not 64
```suggestion
            "toUInt8", [self.column(TRANSACTION_STATUS_ALIAS)], TRANSACTION_STATUS_ALIAS
```

Reply:
Good call